examples/blinky-freertos-ch58x/platformio.ini

```diff
@@ -21,3 +21,6 @@ upload_protocol = isp
 
 [env:genericCH582F]
 board = genericCH582F
+
+[env:genericCH592F]
+board = genericCH592F
```

Comment:
If the example is now valid for ch58x and ch59x, then it should be possibly renamed (currently `blinky-freertos-ch58x`)

Reply:
Should I rename it as blinky-freertos-ch5xx?

Reply:
Mhm sadly that does not cover CH57x and CH56x,  so i'd rather have `blinky-freertos-ch58x-ch59x`.

Reply:
ok. done